### PR TITLE
Update Dialog from citra to yuzu

### PR DESCRIPTION
**Pull request by N00byKing**
Just updates the Messages to say to reference Yuzu and the Switch instead of Citra and the 3DS

### CODE REVIEW

**Comment by N00byKing** [comment]
why though

File: src/yuzu/main.cpp [hunk 1]
```diff
@@ -673,18 +673,18 @@ void GMainWindow::UpdateStatusBar() {
...
-           "href='https://citra-emu.org/wiki/"
-           "dumping-system-archives-and-the-shared-fonts-from-a-3ds-console/'>Dumping System "
-           "Archives and the Shared Fonts from a 3DS Console</a>.<br/><br/>Would you like to quit "
-           "back to the game list? Continuing emulation may result in crashes, corrupted save "
-           "data, or other bugs.");
+    const QString common_message = tr(
+        "The game you are trying to load requires additional files from your Switch to be dumped "
+        "before playing.<br/><br/>For more information on dumping these files, please see the "
+        "following wiki page: <a "
+        "href='https://citra-emu.org/wiki/"
```
Comment (bunnei) [inline src/yuzu/main.cpp]: https://yuzu-emu.org/wiki/ ?

File: src/yuzu/main.cpp [hunk 2]
```diff
@@ -673,18 +673,18 @@ void GMainWindow::UpdateStatusBar() {
...
+        "dumping-system-archives-and-the-shared-fonts-from-a-3ds-console/'>Dumping System "
```
Comment (bunnei) [inline src/yuzu/main.cpp]: We should change this to link to a placeholder wiki page for yuzu/switch (even if it's empty for now). Feel free to ask on Discord if you have any questions about how to create this

Comment (N00byKing) [reply src/yuzu/main.cpp]: Will do (and if needed ask), but tomorrow. A fuze just blew and I am out replacing it
(kill me)

Comment (N00byKing) [reply src/yuzu/main.cpp]: now waiting for the website to update

**Comment by mailwl** [comment]
for me - useless dialog for yuzu. Just burn it with fire

**Comment by N00byKing** [comment]
I have updated the Link to point to the placeholder Pages. Be aware that the Links are dead right now until the Travis Bot updates.
One thing to note is that dumping installed titles is possible right now, and, as such, I think someone should write the page. I have no experience in this, and my English is horrible, which is why I just made it a placeholder. However, If you want I can try and write the Page.

@mailwl why do you think that its useless?

**Comment by mailwl** [comment]
@N00byKing, forget, i was wrong

**Comment by N00byKing** [comment]
Links to Placeholder Pages "Dumping Game Cartridges" and "Dumping Installed Titles" now Online

**Comment by bunnei** [comment]
lgtm, ty 👍

**Pull request merged**